content/introduction/_index.md

```diff
@@ -1,60 +1,187 @@
+If you notice something missing or have other ideas for improving this documentation, 
+please [reach out to let us know][help_improve_docs].
 
-## Community
+### Data transmission
 
-Get in touch with other members of the libp2p community who are building tools and applications with libp2p! You can ask questions, discuss new ideas, or get support for problems at [libp2p discussion forums](https://discuss.libp2p.io/), but you can also [hop on IRC](/community/irc/) for a quick chat.
+The transport layer is at the foundation of libp2p, which is responsible for 
+the actual transmission and receipt of data from one peer to another. There are many 
+ways to send data across networks in use today, with more in development and still more yet 
+to be designed. 
 
-See the other links in the community section for more information about meetings, events, apps people are building, and more.
+libp2p provides a simple [interface](https://github.com/libp2p/js-libp2p-interfaces) 
+that can be adapted to support existing and future protocols, allowing libp2p applications 
+to operate in many different runtime and networking environments.
 
-Information about contributing to libp2p and about other software projects in the community are also hosted here.
+### Peer identity
 
+Knowing who you're talking to is key to secure and reliable communication in a world 
+with billions of networked devices. libp2p uses 
+[public key cryptography](https://en.wikipedia.org/wiki/Public-key_cryptography) 
+as the basis of peer identity, which serves two complementary purposes.
 
-### Get Involved
+1. It gives each peer a globally unique "name", in the form of a 
+   [PeerId][definition_peerid]. 
+2. The `PeerId` allows anyone to retrieve the public key for the identified 
+   peer, which enables secure communication between peers.
 
-libp2p is an open-source community project. While [Protocol Labs](https://protocol.ai) is able to sponsor some of the work around it, much of the design, code, and effort is contributed by volunteers and community members like you. If you’re interested in helping improve libp2p, check the [contributing](/contributing/) guide to get started.
+### Secure communication
 
-If you are diving in to contribute new code, make sure you check both the [contribution guidelines](https://github.com/libp2p/community/blob/master/CONTRIBUTE.md) and the style guide for your language ([Go](https://github.com/ipfs/community/blob/master/CONTRIBUTING_GO.md), [JavaScript](https://github.com/ipfs/community/blob/master/CONTRIBUTING_JS.md)).
+We must be able to send and receive data between peers securely, 
+where we can trust the [identity](#peer-identity) of the peer we're 
+communicating with no third party can read our conversation or alter it in flight.
 
+libp2p supports *upgrading* a connection provided by a [transport protocol](#transport) 
+into a securely encrypted channel. The process is flexible and supports multiple 
+methods of encrypting communication. 
```

Comment:
Better: All libp2p connections are encrypted and authenticated. Some transport protocols are encrypted at the transport layer (e.g. QUIC), for other protocols libp2p runs a cryptographic handshake on top of an unencrypted connection (e.g. TCP).

Reply:
Adapted